### PR TITLE
Add load test logic

### PR DESCRIPTION
**Pull request by wanghuan-520**
Add load test logic

### CODE REVIEW

File: global.json
Comment (zhifenglee-aelf) [inline global.json:1]: This file should not be committed

**Review by zhifenglee-aelf — CHANGES_REQUESTED** [review]

File: src/AISmart.Application.Grains/Agents/MockA/MockAGAgent.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+        State.ThreadIds.Add(eventData.Id);
+
+        var publishEvent = new MockBThreadCreatedEvent
+        {
+            Content = $"A Thread {eventData.Content} has been published."
+        };
+
+        await PublishAsync(publishEvent);
+        await PublishAsync(new RequestAllSubscriptionsEvent());
```
[the remark this answers is not in the document]
Comment (wanghuan-520) [reply src/AISmart.Application.Grains/Agents/MockA/MockAGAgent.cs]: Modified

File: src/AISmart.Application.Grains/Agents/MockB/MockBGAgent.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+        State.ThreadIds.Add(eventData.Id);
+
+        var publishEvent = new MockCThreadCreatedEvent
+        {
+            Content = $"A Thread {eventData.Content} has been published."
+        };
+
+        await PublishAsync(publishEvent);
+        await PublishAsync(new RequestAllSubscriptionsEvent());
```
[the remark this answers is not in the document]
Comment (wanghuan-520) [reply src/AISmart.Application.Grains/Agents/MockB/MockBGAgent.cs]: Modified

File: global.json
Comment (wanghuan-520) [reply global.json:1]: Deleted

**Review by zhifenglee-aelf — APPROVED** [review]
Please Squash and Merge so as to revert easily.

**Pull request merged**